components/TextControls/examples/MDCTextControlTextFieldTypicalUseExample.m

```diff
@@ -48,33 +48,39 @@ @implementation MDCTextControlTextFieldTypicalUseExample
 - (void)viewDidLoad {
   [super viewDidLoad];
   self.title = kExampleTitle;
-
   if (!self.containerScheme) {
     self.containerScheme = [[MDCContainerScheme alloc] init];
   }
 
+  self.view.backgroundColor = self.containerScheme.colorScheme.backgroundColor;
+  if (@available(iOS 13.0, *)) {
+    self.view.backgroundColor = [UIColor systemBackgroundColor];
```

Comment:
I think you should be using the scheme background color. The scheme may be dynamic or not, and that would be intentional, right?

Reply:
Done